Dockerfile

```diff
@@ -58,9 +58,7 @@ RUN source activate cudf && \
     cmake .. -DCMAKE_INSTALL_PREFIX=${CONDA_PREFIX} && \
     make -j install && \
     make python_cffi && \
-    make install_python && \
-    cd python && \
-    python setup.py install
+    make install_python
```

Comment:
This change is good. No longer redundant. But note that make python_cffi and make install_python are separate commands -- we need to keep it that way.